### DISCUSSION TITLE
CALDAV SELFHOSTED

### DISCUSSION THREAD

**Discussion by jonzarmar14** [FossifyOrg/General-Discussion#460] [Q&A]
I was trying to use a self hosted caldav server to avoid using google but I want to sync only google accounts appear. Is there any way to achieve this?

**Comment by Aga-C** [comment]
What app for CalDAV synchronization are you using on your phone?

**Reply by jonzarmar14** [reply]
I was trying to do it directly on fossify, but I read that the app can't do it directly as is does not have internet access. So now im using DavX5
